.github/workflows/pr-merge-webhook.yml

```diff
@@ -10,20 +10,19 @@ on:
 
 jobs:
   create-docs-ticket:
```

Comment:
This job name should be changed.

Theoretically, if we were pushing webhooks to multiple places, I think we would create a new _job_ (as opposed to adding new a _stop_ to this job).

Given that the contents of the run are Jira specific (ie, X-Automation-Webhook-Token), I think it makes sense to set the *job* name to something jira specific. `jira-webhook-pr-merge`? (as opposed to something more general like `webhook-pr-merge`)

Reply:
This also ensures we have immediate context in the case of failure:

> "message": "${{ github.workflow }}/${{ github.job }} failure"